src/host_capabilities/verification.rs

```diff
@@ -62,6 +62,51 @@ pub fn verify_keyless_exact_match(
     verify(req)
 }
 
+/// verify sigstore signatures of an image using keyless. Here, the provided
+/// subject string is streated as a URL prefix, and sanitized to a valid URL on
+/// itself by appending `\` to prevent typosquatting. Then, the provided subject
```

Comment:
```suggestion
/// itself by appending `/` to prevent typosquatting. Then, the provided subject
```

URLs end with `/`, not `\`